tests/default/security/api/nodesdn.yaml

```diff
@@ -1,65 +0,0 @@
-$schema: ../../../../json_schemas/test_story.schema.yaml
```

Comment:
Why are we removing this story?

Reply:
See https://github.com/opensearch-project/opensearch-api-specification/issues/654, the story doesn't test anything.